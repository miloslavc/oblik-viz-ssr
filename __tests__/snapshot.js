import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../pages/index';

jest.mock('next/link', () => {
  return ({ children }) => {
    return children;
  };
});

it('renders homepage unchanged', () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});
