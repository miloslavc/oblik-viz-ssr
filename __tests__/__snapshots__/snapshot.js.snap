// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
Array [
  <header>
    <nav>
      <a>
        Home
      </a>
      |
      <a>
        Contact
      </a>
    </nav>
  </header>,
  <main>
    <div
      className="pages__Container-bumurp-0 kjqZjY"
    >
      <h1
        className="pages__Title-bumurp-1 glEVRL"
      >
        Welcome to
         
        <a
          href=""
          onClick={[Function]}
        >
          Kolrs!
        </a>
      </h1>
    </div>
  </main>,
  <footer>
    Â© 
    2020
    , All rights reserved. 
  </footer>,
]
`;
